agreements/expenses-agreement.md

```diff
@@ -0,0 +1,44 @@
+# Expenses Agreement
+
+As a worker-owned business Root Systems members all have the ability to claim expenses relating to the operation of the business. 
+
+## This may include:
+
+ - Technical equipment (laptops, keyboards, etc...)
+ - Office equipment (coffee machine, chairs, desk, lights, etc...)
+ - Office space rent
+ - Retreats and Away days (travel, food and accomadation)
+ - Event expenses (tickets, foods, etc...)
+ - Every day business lunches (50% tax deductable for members, 100% for guests)
+ - Travel expenses (transport, hotels, flights, etc...)
+ - Saas tools (google, toggl, loomio, xero, etc...)
+ - Memberships (Enspiral)
+
+## Accounting for expenses
+
+Expenses fall into two categories:
+
+ - Team expense
+ - Member expense
+
+### Team expense
+
+These are expenses that are accounted for out of core funds and facilitated by the Coordinators. The list at the bottom of this agreement defines ongoing Team expenses that we do not need raise a decision for. Beyond this any member can raise a proposal via Loomio to add another Team expense (one-off or on-going). Team expenses will be accounted for out of our opperational budget in Xero.
+
+### Member expense
+
+Any Member can expense off any items they identify as being a business expense. Coordinators will facilitate this to provide support and understanding of what constitutes a business expense. Member expenses will be accounted for through deducting the expense minus GST from their internal account.
+
+**Everyday business lunch expenses will be standardised to $20/day to make internal accounting easier.
```

Comment:
So I am suggesting lunches not a team expense but a member expense. Hence why its in the members section. We can tidy up the text so thats clearer.

We could also drop it to 15